### ISSUE TITLE
Cant set dns1 address. "some but not all values in the same group of inclusion 'dns'."

### ISSUE THREAD

**Issue by pixiandreas** [esphome/ESPHome#321]
**Operating environment (Hass.io/Docker/pip/etc.):**
pip

**ESP (ESP32/ESP8266/Board/Sonoff):**
esp8266 wemod d1

**Affected component:**
wifi

**Description of problem:**

**Problem-relevant YAML-configuration entries:**
```yaml

   manual_ip:
     static_ip: 192.168.50.101
     gateway: 192.168.50.1
     subnet: 255.255.255.0
     dns1: 192.168.50.1

```

**Traceback (if applicable):**
config error: 
  
  some but not all values in the same group of inclusion 'dns'.
  manual_ip:  [source test.yaml:15]
    static_ip: 192.168.50.101
    gateway: 192.168.50.1
    subnet: 255.255.255.0
    dns1: 192.168.50.1
```

```

**Additional information:**

The config check complain about the dns1
Without dns1 it works as it should.

**Comment by OttoWinter** [comment]
The error means you need to specify both `dns1` and `dns2`

**Comment by nacbooth** [comment]
I had the same problem and discovered that DNS2 had to be set as well; I simply use the same IP address for both (as I only have one [practical] DNS server on my network.

**Comment by brandond** [comment]
This should be fixed in #357 which hasn't yet been merged to dev.
